pom.xml

```diff
@@ -19,7 +19,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.jongo</groupId>
+    <groupId>org.evergage.thirdparty.jongo</groupId>
```

Comment:
I think we should be consistent by using `com.evergage.thirdparty.*`, since we are already authorized to publish to `com.evergage.*`.

Reply:
Makes sense. Thanks for the eyes. Fixing...